scripts/styleguide.config.js

```diff
@@ -1,4 +1,6 @@
 const path = require('path');
+// const typescriptDocGen = require('react-docgen-typescript');
```

Comment:
Do we need these commented lines? Same for the exports below.

Reply:
They are currently left there for reference/documentation. Will be nuked eventually.